Docs/features/task-search.md

```diff
@@ -0,0 +1,23 @@
+#### Task Search
+
+As of 0.4.12, Singularity has better support for searching historical tasks. A global task search endpoint was added:
+
+`/api/history/tasks` -> Retrieve the history sorted by startedAt for all inactive tasks.
+
+... and the above endpoint as well `/api/history/request/{requestId}/tasks` now take additonal query parameters:
+
+- `requestId`: Optional request id to match (only for `/api/history/tasks` endpoint as it is already specified in the path for `/request/{requestId}/tasks`)
+- `deployId`: Optional deploy id to match
+- `host`: Optional host (slave host name) to match
+- `lastTaskStatus`: Optional [`ExtendedTaskState`](../reference/api-docs/models.md#model-ExtendedTaskState) to match
+- `startedAfter`: Optionally match only tasks started after this time (13 digit unix timestamp)
+- `startedBefore`: Optionally match only tasks started before this time (13 digit unix timestamp)
+- `orderDirection`: Sort direction (by `startedAt`), can be ASC or DESC, defaults to DESC (newest tasks first)
+- `count`: Maximum number of items to return
+- `page`: Page of items to view (e.g. page 1 is the first `count` items, page 2 is the next `count` items)
```

Comment:
Count and page are optional too. Page defaults to 1, and I believe count defaults to infinite but I don't remember exactly.